projects/control-service/projects/pipelines_control_service/src/integration-test/resources/application-private-builder.properties

```diff
@@ -0,0 +1,3 @@
+datajobs.builder.registrySecret=integration-test-docker-pull-secret
+datajobs.builder.registrySecret.content.testOnly=${BUILDER_TEST_REGISTRY_SECRET}
+datajobs.builder.image=vmwaresaas.jfrog.io/taurus-dev/versatiledatakit/job-builder:1.2.3
```

Comment:
We certainly cannot (and should not) use VMware repository in the open source project. 

There's one for the CICD - https://github.com/vmware/versatile-data-kit/wiki/Gitlab-CICD
 
 It would be better if we do not rely on external network dependencies but start docker ourselves though .

Reply:
So start a private docker instance inside the CICD/local cluster for integration tests?

Reply:
I agree

Reply:
> So start a private docker instance inside the CICD/local cluster for integration tests? 

That's what I think would be best long term. 

But we use already use external git and docker repositories for the integration tests (based on github - git and ghcr), it's more proper as part of this PR to re-use them. Any bigger change (like starting our won docker repo) should be done in separate PR (they are documented at https://github.com/vmware/versatile-data-kit/wiki/Gitlab-CICD)

Reply:
Done.

Reply:
> 

> > So start a private docker instance inside the CICD/local cluster for integration tests?
> 
> That's what I think would be best long term.

Actually, what I mean is the test needs to spawn and destroy what it needs during test runtime. So using something like https://www.testcontainers.org would be great.